src/Development/Shake/Internal/Rules/File.hs

```diff
@@ -371,8 +371,13 @@ fileForward help act = addUserRule $ FileRule help $ fmap ModeForward . act
 --     'Development.Shake.cmd' \"rot13\" [src] \"-o\" [out]
 -- @
 --
---   Usually @need [foo,bar]@ is preferable to @need [foo] >> need [bar]@ as the former allows greater
---   parallelism, while the latter requires @foo@ to finish building before starting to build @bar@.
+--   Note that the following expressions are all equivalent. @foo@ and @bar@ are built in parallel:
+--
+--   - @need [foo,bar]@
+--   - @need [foo] '*>' need [bar]@
+--   - @need [foo] '>>' need [bar]@
+--
+--   In this situation, @need [foo, bar]@ is preferable, since the parallelism is the most obvious.
```

Comment:
I had also wanted to add a note here saying something like:

> If you do require that two separate `need`s are built sequentially, you can put another (non-`need`) `Action` in between, like the following:
>
> ```haskell
> do
>   need [foo]
>   liftIO (putStr "")
>   need [bar]
> ```

However:

- I wasn't sure what would be the best way to write this (`liftIO putStr` seems kind of hacky)
- This isn't something that _just_ affects 'need', it also affects all the other actions that use `need` internally, like `readFileLines`.